server/src/test/java/org/candlepin/sync/ConsumerExporterTest.java

```diff
@@ -70,7 +70,8 @@ public void testConsumerExport() throws IOException {
         json.append("\"urlWeb\":\"/subscriptions\",");
         json.append("\"urlApi\":\"/candlepin\",");
         json.append("\"contentAccessMode\":\"access_mode\"}");
-        assertTrue(TestUtil.isJsonEqual(json.toString(), writer.toString()));
+        assertTrue(json.toString() + "\n" + writer.toString(),
```

Comment:
Nitpick: these changes are good, but are unrelated to the changes necessary for Jackson... Separate PR please?